content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,144 @@
-   Marketplace`. This is the original sign-up marketplace of your seller account (e.g. if you
-   created your account on Amazon Germany, your sign-up marketplace will be `amazon.de`)
+Then, return to :menuselection:`Sales app --> Configuration --> Settings --> Connectors section`,
+and click on the :guilabel:`Amazon Accounts` link.
 
-#. Upon saving, a button to :guilabel:`link with Amazon` appears. Click on it to be redirected to
-   the Amazon login page, or directly to the consent page if you are already logged in. There,
-   confirm that you want Amazon to give Odoo access to your account and related data.
+.. image:: setup/amazon-accounts-link-setting.png
+   :align: center
+   :alt: The Amazon Accounts link beneath the Amazon Synch settings in Odoo Sales.
 
-#. Amazon redirects you to Odoo, with your account registered.
+Doing so reveals a separate :guilabel:`Amazon Accounts` page. From here, click :guilabel:`New` to
+create, and subsequently link, a new Amazon account.
 
-Once the account is registered, the marketplaces available to this account are synchronized and
-listed under the :guilabel:`Marketplaces` tab. If you wish, you can remove some items from the list
-of synchronized marketplaces to disable their synchronization.
+On the blank :guilabel:`Amazon Account` form page, start by choosing a name for the account (e.g.
+`American Marketplace`). Then, in the :guilabel:`Credentials` tab, select the desired marketplace on
+which the seller account was initially created from the :guilabel:`Home Marketplace` drop-down menu.
+
+.. image:: setup/amazon-accounts-form-page.png
+   :align: center
+   :alt: A typical Amazon Account form page in the Odoo Sales application.
+
+.. note::
+   After validation of credentials, the marketplaces to which this account has access is
+   synchronized, and automatically made available.
+
+After saving, the field in the :guilabel:`Credentials` tab is replaced by a :guilabel:`Link with
+Amazon` button.
+
+.. image:: setup/amazon-accounts-form-link-button.png
+   :align: center
+   :alt: A typical Amazon Account form page and Link with Amazon button in Odoo Sales.
+
+Clicking that button redirects to either the Amazon login page, or directly to the required consent
+page, if the user is already logged in to Amazon.
+
+On this page, confirm that Amazon is allowed to give Odoo access to the account and related data.
+
+Upon confirmation, Amazon returns the user to Odoo, and the account has been registered.
+
+With the account successfully registered, the marketplaces available to this specific account are
+synchronized and listed under the :guilabel:`Marketplaces` tab.
+
+If desired, remove items from the list of synchronized marketplaces to disable synchronization.
 
 Match database products in Amazon
 =================================
 
 When an Amazon order is synchronized, up to three sales order items are created in Odoo for each
```

Comment:
I totally understand where you're coming from. I always read it as up to 3 line items being added to the sales order in Odoo, to represent the 3 potential charges coming from the Amazon purchase. As you know, I was only able to work off context clues provided in the previous version of this doc, because I don't have access to a paid-for Amazon Seller account. However, since it was previously "OK'd" by the PO/PE, I think I just need to clarify this sentence a bit more (which I just took care of :) ) -- I believe the newly restructured sentence spells things out a bit clearer. That said, if you think it should be changed even further, I'm always open to suggestions! Thanks for pointing this out!